test/e2e/memcached_test.go

```diff
@@ -240,16 +240,18 @@ func TestMemcached(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Command 'dep ensure' failed: %v\nCommand Output:\n%v", err, string(cmdOut))
 	}
-	// link local sdk to vendor if not in travis
+	// Copy local sdk to vendor if not in CI.
 	if repo == "" {
-		for _, dir := range []string{"pkg", "internal"} {
+		for _, dir := range []string{filepath.Join("internal", "pkg")} {
```

Comment:
I only copy `internal/pkg` here because `internal/util` has dependencies that `memcached-operator` does not satisfy, which causes a build error. This is a temporary workaround until #1001 is merged, since `go mod` can resolve non-`master` commits.